devenv.local.nix

```diff
@@ -0,0 +1 @@
+{ pre-commit.default_stages = ["manual"]; }
```

Comment:
I assume this wasn't meant to be committed

Reply:
Oh, yeah, I removed it from main already. Looks like it crept in here.